core/unit_test/TestAtomicViews.hpp

```diff
@@ -1392,9 +1392,12 @@ TEST(TEST_CATEGORY, atomic_views_integral) {
     ASSERT_TRUE(
         (TestAtomicViews::AtomicViewsTestIntegralType<int64_t, TEST_EXECSPACE>(
             length, 4)));
+#ifndef KOKKOS_ENABLE_OPENACC
```

Comment:
Same question about guarding in `AtomicViewsTestIntegralType` instead.

Reply:
Changed the way to guard unsupported atomic tests for the OpenACC backend.